### DISCUSSION TITLE
Plotting performance issue

### DISCUSSION THREAD

**Discussion by cexmad** [Chia-Network/chia-blockchain#5447] [Support]
Hi,

I got question for improve plotting speed. Do I need to change any options before start plotting?
While the CPU is being used 100% during the plotting process, I cannot get the same performance from the nvm-e ssd.

You can find my hardware features below;
Nvm-e ssd: 5000mb/s read- 3000mb/s write
CPU: 1.90hz 12 core with 24 logical processor.

Plotting options for 1x or 10x parallel plotting:
Ram: 50gb
Threads: 20

Should I give less threads to process for increase drive using capacity? Before nvm-e, I was able to create one plot using the normal ssd with 500mb/s speed in 9 hour. 

Tnx for your reply.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
